phoenix-core/src/main/java/org/apache/phoenix/util/DateUtil.java

```diff
@@ -190,6 +193,68 @@ private static long parseDateTime(String dateTimeValue) {
         return ISODateFormatParser.getInstance().parseDateTime(dateTimeValue);
     }
 
+    private static long getLongInJulianChrono(long ts) {
+        DateTime dt1 = new DateTime(ts, GJChronology.getInstanceUTC());
+        DateTime dt2 = new DateTime(ts, ISOChronology.getInstanceUTC());
+        Date d1 = parseDate(dt1.toString());
+        Date d2 = parseDate(dt2.toString());
+        long l1 = d1.getTime();
+        long l2 = d2.getTime();
+        return (ts + (l1 - l2));
```

Comment:
Any chance you could include a description as to how/why this math works?

Reply:
I made it a bit simpler and added comments in the latest change.